packages/components/src/components/table/table-row/table-row-styles.ts

```diff
@@ -6,17 +6,30 @@ import {
   hostHiddenStyles,
   hoverMediaQuery,
 } from '../../../styles';
+import { borderRadiusSmall } from '@porsche-design-system/utilities-v2';
+import type { Theme } from '../../../types';
 
-export const getComponentCss = (): string => {
+export const offset = '-2px';
+export const getComponentCss = (theme: Theme): string => {
   return getCss({
     '@global': {
       ':host': addImportantToEachRule({
+        position: 'relative',
         display: 'table-row',
         ...hostHiddenStyles,
         ...hoverMediaQuery({
-          transition: getTransition('background-color'),
-          '&(:hover)': {
-            backgroundColor: getThemedColors('light').backgroundSurfaceColor,
+          '&::before': {
+            content: '""',
+            position: 'absolute',
+            left: offset,
```

Comment:
Please use `getInsetJssStyle()` like everywhere else.

Reply:
I didn't know that we have such a tool, fixed it